guides/common/modules/proc_migrating-to-external-databases.adoc

```diff
@@ -44,19 +37,31 @@ PGPASSWORD='_Pulpcore_Password_' pg_restore -h _postgres.example.com_ -U pulp -d
 [options="nowrap", subs="+quotes,attributes"]
 ----
 {installer-scenario} \
-    --foreman-db-host _postgres.example.com_ \
-    --foreman-db-password _Foreman_Password_ \
-    --foreman-db-database foreman \
-    --foreman-db-manage false \
-    --foreman-db-username foreman \
+    --katello-candlepin-manage-db false \
     --katello-candlepin-db-host _postgres.example.com_ \
     --katello-candlepin-db-name candlepin \
-    --katello-candlepin-db-password _Candlepin_Password_ \
-    --katello-candlepin-manage-db false \
     --katello-candlepin-db-user candlepin \
+    --katello-candlepin-db-password _Candlepin_Password_ \
     --foreman-proxy-content-pulpcore-manage-postgresql false \
     --foreman-proxy-content-pulpcore-postgresql-host _postgres.example.com_ \
     --foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
+    --foreman-proxy-content-pulpcore-postgresql-user pulp \
     --foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_ \
-    --foreman-proxy-content-pulpcore-postgresql-user pulp
+    --foreman-db-manage false \
+    --foreman-db-host _postgres.example.com_ \
+    --foreman-db-database foreman \
+    --foreman-db-username foreman \
+    --foreman-db-password _Foreman_Password_
+----
+. Remove the PostgreSQL package on {ProjectServer}.
```

Comment:
Does the service need to be stopped before this? I *think* package scripts will do that for us, but might be worth doing it explicitly?

Reply:
Also, add a step to verify that the app still works while the DB is down (but not removed yet)?

Reply:
A step to stop the service, then verify it still works and then removes it makes sense to me.

How would we verify it? I think `foreman-maintain` health check would flag the stopped PostgreSQL service as down, while it shouldn't be used. `hammer ping` perhaps?

Reply:
`hammer ping` seems reasonable.
(I would expect `f-m health check` to also work, as it should detect that the app is now using a remote DB, but untested)

Reply:
I just made an assumption based, but I can't easily figure it out based on reading the `foreman-maintain` source code.